### ISSUE TITLE
Screen reader not reading some full keybindings

### ISSUE THREAD

**Issue by vsccarl** [Microsoft/vscode#6140]
- VSCode Version: 1.1.0-alpha
- OS Version: Windows 10

Steps to Reproduce:

Prereq: Install NVDA

There are a few keybindings not being read:
1. Right click in a file and hover over "Change All Occurances" reads as "Ctrl +" and does not read F2
2. In the Goto menu "Go to Definition" does not read F12.
3. In the File menu "Close Editor" reads "Ctrl +" and does not read F4
4. In the View menu "Zoom out" reads "Ctrl +" and does not read "-"
5. In the View menu "Split Editor" reads "Ctrl +" and does not read "\"

**Comment by bpasero** [comment]
@vsccarl are other keybindings being read fully?

**Comment by vsccarl** [comment]
@bpasero yes, all others are being read fully.

**Comment by bpasero** [comment]
We will change to custom menus on Windows. For now you can set `"window.titleBarStyle": "custom"`.
